libevm/VM.cpp

```diff
@@ -115,6 +113,22 @@ void VM::adjustStack(unsigned _removed, unsigned _added)
 #endif
 }
 
+void VM::sstoreGas()
+{
+	if (m_ext->staticCall)
```

Comment:
I copied the existing code in SSTORE exactly.  Could move this one piece to SSTORE and to XSTORE?